server/insights/src/main/java/org/infinispan/server/insights/logging/Log.java

```diff
@@ -43,17 +40,21 @@ public interface Log extends BasicLogger {
    @Message(value = "Remote Insights reporting", id = 32005)
    void insightsEnabled();
 
-   @Message(value = "Error configuring Red Hat Insight client", id = 32006)
-   CacheConfigurationException insightsConfigurationError();
+   @LogMessage(level = INFO)
+   @Message(value = "The environment variable %s or the system property %s cannot be set to true in order to " +
+         "activate the Insights client service. The remote insights reporting will be disabled.", id = 32006)
+   void unsupportedInsightsConfigurationProperty(String systemProperty, String environmentVariable);
```

Comment:
Just a nitpick but I find the naming of the method a bit confusing. Is the property unsupported?

Reply:
I think so. In the sense that if the user has it in their system properties or in their environment variable is to set somehow the configuration of the Insights client.

Reply:
I tried to simplify it a bit.